CMakeLists.txt

```diff
@@ -94,6 +97,7 @@ set(TRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION "" CACHE STRING "ONNXRuntime OpenV
 set(TRITON_BUILD_CUDA_VERSION "" CACHE STRING "Version of CUDA install")
 set(TRITON_BUILD_CUDA_HOME "" CACHE PATH "Path to CUDA install")
 set(TRITON_BUILD_CUDNN_HOME "" CACHE PATH "Path to CUDNN install")
+set(TRITON_BUILD_PLATFORM "" CACHE STRING "Platform of build")
```

Comment:
I was wondering why do we need to explicitly specify this flag? What would `platform.system().lower()` return on jetson? It seems we were able to handle Windows and x86 without this flag.

Reply:
If you run `platform.system().lower()` on Jetson (bare metal or Docker), it'll return "linux." That's insufficient for knowing whether we're running on Jetson or not.

Reply:
How about using a combination of `platform.system()` and `platform.machine()`?

Reply:
That would work. Great find. Are you okay with me creating a small Jira ticket to refactor these few lines when I get back from vacation? I'm trying to get the move to Docker out within the next few hours, so there's not enough time for another CI run with the limited number of Jetson runners.

Reply:
Sounds good. I'll resolve this thread.

Reply:
Thanks, Iman! Created ticket here: https://jirasw.nvidia.com/browse/DLIS-4883